src/main/java/segelzwerg/sporttooolbox/iunits/distance/Distance.java

```diff
@@ -10,7 +10,11 @@
 public interface Distance {
     double MILES_TO_YARDS = 1760.0;
     double FATHOMS_TO_NAUTICAL_MILES = 1 / 1013.3333334;
+    float MILES_TO_KM = 1.6093444f;
+    float KILOMETER_TO_METERS = 1000f;
 
+    float NAUTICAL_TO_KM = 1.852f;
+    float NAUTICAL_TO_FATHOMS = 1012.685914261f;
```

Comment:
I think it's better to have these variables inside the classes because `Miles` don't need to know about `Kilometer's` conversion.

Reply:
Yeah, I moved them for an example in this task. As for me, the better solution is to have a service that would convert distance instances to each other. I think I'll show you my suggestions for the code this week.